modules/slider/LockedSlider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { withStyles } from "@diana-ui/base";
+import { ThemeStyleSheetFactory, WithStylesProps } from "@diana-ui/types";
+import { Icon } from "@diana-ui/icon";
+import Slider, { ISliderProps } from "./Slider";
+
+export interface ILockedSliderProps extends ISliderProps {
+  startLocked?: boolean;
+  lockedIcon: string | React.ReactNode;
```

Comment:
Could we just choose one way of having it? seems confusing to have for example lockedIcon as string and unlockedIcon as Icon element